### PR TITLE
add customid ref to poc2prod

### PR DESCRIPTION
**Pull request by vmadhusudanan**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://62cf6087fca250614a27219c--cozy-fox-0defec.netlify.app

File: docs/guides/production.mdx [hunk 1]
```diff
@@ -35,14 +35,18 @@ You should make sure [metrics](https://docs.statsig.com/metrics) and dimensions
 ![image](https://user-images.githubusercontent.com/31516123/178082121-bb258851-29e5-4732-bd2a-af87833f7570.png)
 
 
-## 5. Set up integrations
+## 5. Decide what IDs to experiment on
+
+Most experiments or feature rollouts on Statsig target users (identified by userID). Another identity type Statsig supports is stableID. This is a Statsig client SDK managed ID unique to a device - typically used to experiment on anonomyous users. You can also create arbitrary units of assignment (eg. organizationID for B2Bs, VIN or vehicleIDs if you're controlling features on cars, creatorID if you have a two sides marketplace with creators and consumers (regular users) and want to target both). [Read more](https://docs.statsig.com/guides/experiment-on-custom-id-types).
```
Comment (ritvik-statsig) [inline docs/guides/production.mdx:40]: nit: I found this a little hard to follow, I would maybe split this into the three cases with bullet points: userID, stableID (for cases where you want to experiment on users who do not have a userID, such as anonymous or logged out users), other custom IDs (the examples you mentioned)

or if you don't like that, then at least anonomyous -> anonymous :P

**Review by ritvik-statsig — APPROVED** [review]
